FFXIVClientStructs/FFXIV/Client/UI/Info/InfoProxyItemSearch.cs

```diff
@@ -108,15 +109,16 @@ public unsafe partial struct MarketBoardListing {
 
     [FieldOffset(0xA8)] public bool IsHqItem;
     [FieldOffset(0xA9)] public byte MateriaCount;
+    [FieldOffset(0xAA)] public bool IsMannequin;
 
     // [FieldOffset(0xAC)] public ushort Unk_0xAC;
 
     /// <summary>
     /// The Town (from EXD) that this marketboard entry is from.
     /// </summary>
     [FieldOffset(0xB0)] public byte TownId;
-
-    [FieldOffset(0xB1)] public byte StainId;
+    [FieldOffset(0xB1)] public byte StainId1;
+    [FieldOffset(0xB2)] public byte StainId2;
```

Comment:
For consistency, please rename this to:
```suggestion
    [FieldOffset(0xB1)] public byte Stain1Id;
    [FieldOffset(0xB2)] public byte Stain2Id;
```